### PR TITLE
BC-7279 - Integrate MediaShelf changes into new Sidebar

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Short Description
Change sidebar according to feedback (see "Changes" section)

## Links to Ticket and related Pull-Requests
BC-7279

## Changes
- Add "feature" and "featureValue" property to sidebar items (see media shelf PRs)
- Add media-shelf route as item
- Remove A11Y Category
- Add A11Y Items to System Category
- Move "What's new?" item to System category
- Add instance name at the bottom (see images)
- Change wording to "Datenschutzhinweise"
- Remove "Trainings" section

## Checklist before merging

- [ ] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/modules/ui/layout/sidebar/Sidebar.vue [hunk 1]
```diff
@@ -20,27 +20,37 @@
...
-			<div class="py-3">
+			<div class="pt-3">
 				<SidebarItem
 					v-for="link in legalItems"
 					:key="link.title"
 					:item="link"
 				/>
 			</div>
 		</VList>
+		<div data-testid="instanz-sidebar" class="ml-4 pb-3 text-disabled">
```
Comment (odalys-dataport) [inline src/modules/ui/layout/sidebar/Sidebar.vue:31]: Can we name it "sidebar-instance-name" instead?

File: src/modules/ui/layout/sidebar/Sidebar.vue [hunk 2]
```diff
@@ -59,15 +69,27 @@ const userHasPermission = (item: SidebarSingleItem | SidebarGroupItem) => {
...
+				const childHasFeature = hasFeatureEnabled(child);
+
+				return userHasPermission(child) && (!child.feature || childHasFeature);
 			});
 		}
 
-		return userHasPermission(item);
+		const categoryHasFeature = hasFeatureEnabled(item);
+
+		return userHasPermission(item) && (!item.feature || categoryHasFeature);
```
Comment (odalys-dataport) [inline src/modules/ui/layout/sidebar/Sidebar.vue:92]: Are these `!item.feature` and `!child.feature` necessary, if we check for that in the function too?

**Review by odalys-dataport — CHANGES_REQUESTED** [review]

File: src/modules/ui/layout/sidebar/Sidebar.vue (hunk 1 above)
Comment (MartinSchuhmacher) [reply src/modules/ui/layout/sidebar/Sidebar.vue]: changed

File: src/modules/ui/layout/sidebar/Sidebar.vue (hunk 2 above)
Comment (MartinSchuhmacher) [reply src/modules/ui/layout/sidebar/Sidebar.vue]: not necessary indeed, changed that 👍

**Review by odalys-dataport — DISMISSED** [review]

File: src/layouts/newLoggedIn.unit.ts [hunk 3]
```diff
@@ -47,6 +48,9 @@ const setup = () => {
 				[AUTH_MODULE_KEY.valueOf()]: authModule,
 				[ENV_CONFIG_MODULE_KEY.valueOf()]: envConfigModule,
 				[FILE_PATHS_MODULE_KEY.valueOf()]: filePathsModule,
+				[THEME_KEY.valueOf()]: {
+					name: "nbc",
```
Comment (SevenWaysDP) [inline src/layouts/newLoggedIn.unit.ts:52]: FYI we have enum from server SchulcloudTheme

File: src/modules/ui/layout/sidebar/Sidebar.unit.ts [hunk 4]
```diff
@@ -45,6 +46,9 @@ const setup = (permissions?: string[]) => {
 				[AUTH_MODULE_KEY.valueOf()]: authModule,
 				[ENV_CONFIG_MODULE_KEY.valueOf()]: envConfigModule,
 				[FILE_PATHS_MODULE_KEY.valueOf()]: filePathsModule,
+				[THEME_KEY.valueOf()]: {
+					name: "default",
```
Comment (SevenWaysDP) [inline src/modules/ui/layout/sidebar/Sidebar.unit.ts:50]: FYI we have enum from server SchulcloudTheme

File: src/layouts/newLoggedIn.unit.ts (hunk 3 above)
Comment (MartinSchuhmacher) [reply src/layouts/newLoggedIn.unit.ts:52]: good point, also used it in the actual components so why not here...will fix that :)

File: src/modules/ui/layout/sidebar/Sidebar.unit.ts (hunk 4 above)
Comment (MartinSchuhmacher) [reply src/modules/ui/layout/sidebar/Sidebar.unit.ts:50]: see above

**Review by odalys-dataport — APPROVED** [review]

**Review by odalys-dataport — APPROVED** [review]

**Pull request merged**